subgraphs/maker-governance/schema.graphql

```diff
@@ -148,15 +170,13 @@ type Delegation @entity {
   amount: BigInt!
 }
 
-type DelegateVotingPowerChange @entity(immutable: true) {
+type ChiefLockedMKRChange @entity(immutable: true) {
```

Comment:
I think `ExecutiveVotingPowerChange` sounds a bit cleaner and more descriptive. `ChiefLockedMKRChange` feels awkward and un-intuitive. Wdyt?

Reply:
nice yeah